OutOfSchool/OutOfSchool.WebApi/Services/ApplicationService.cs

```diff
@@ -967,4 +994,32 @@ private async Task<int> GetApprovedWorkshopAndChild(Application application)
                                                       application.WorkshopId == a.WorkshopId &&
                                                       Application.ValidApplicationStatuses.Contains(a.Status));
     }
+
+    private async Task SendApplicationUpdateStatusEmail(Application application)
+    {
+        LocalizedString subject = application.Status switch
+        {
+            ApplicationStatus.Approved => localizer["Approved!"],
+            ApplicationStatus.Rejected => localizer["Rejected"],
+            ApplicationStatus.AcceptedForSelection => localizer["Accepted for selection!"],
+            _ => localizer["Application status changed"],
+        };
+        var applicationStatusViewModel = new ApplicationStatusViewModel
+        {
+            ApplicationStatus = application.Status,
+            ChildFullName =
+                $"{application.Child.LastName} " +
+                $"{application.Child.FirstName} " +
+                $"{application.Child.MiddleName}".TrimEnd(),
+            ChildGender = application.Child.Gender,
+            WorkshopTitle = application.Workshop.Title,
+
+            // TODO: Think about pass URL from variable
+            WorkshopUrl = $"{hostsConfig.Value.FrontendUrl}/#/details/workshop/{application.WorkshopId}",
```

Comment:
(Possible design issue) This dependency to directly specified `FrontendUrl` could ntroduce too high coupling b/w front and back, could introduce extra complexity for deployment/proxy configuration/etc. Please reconsider its necessity.

Reply:
I added a new `PathToWorkshopDetailsOnFrontend` key for `Hosts` in `appsettings.json` to avoid hardcoding in the path to the workshop details page.
Unfortunately, I don't see another way to create this path.

Reply:
Well, I was more thinikng about getting URL from request (it could be safe enough for authenticated user), but it could complicate other aspects. So, let it be as you've written.

Reply:
Thank you!